modules/theme/src/themes/default/views/card.overrides

```diff
@@ -22,6 +22,7 @@
 ------------------------------*/
 
 .ui.card {
+    overflow: clip;
```

Comment:
This will override the overflow for all the Cards. Is that intended?

Reply:
Yes, it was. Elements positioned inside will affect but couldn't see any issues. Isolated absolutely positioned ones shouldn't affect 🤔